Client/src/Unified/Services/Submitter/Service.cs

```diff
@@ -552,6 +605,81 @@ private void ResultTask()
     return taskId;
   }
 
+  /// <summary>
+  ///   The method submit with One serialized argument that will be already serialized for byte[] MethodName(byte[]
+  ///   argument).
+  /// </summary>
+  /// <param name="methodName">The name of the method inside the service</param>
+  /// <param name="argument">One serialized argument that will already serialize for MethodName.</param>
+  /// <param name="handler">The handler callBack implemented as IServiceInvocationHandler to get response or result or error</param>
+  /// <param name="token">The cancellation token</param>
+  /// <returns>Return the taskId string</returns>
+  public async Task<string> SubmitAsync(string                    methodName,
+                                        object[]                  argument,
+                                        IServiceInvocationHandler handler,
+                                        CancellationToken         token = default)
+  {
+    await semaphoreSlim_.WaitAsync(token);
+
+    var blockRequest = new BlockRequest
+                       {
+                         Payload = new ArmonikPayload
+                                   {
+                                     ArmonikRequestType  = ArmonikRequestType.Execute,
+                                     MethodName          = methodName,
+                                     ClientPayload       = ProtoSerializer.SerializeMessageObjectArray(argument),
+                                     SerializedArguments = false,
+                                   },
+                         Handler = handler,
+                         Lock    = semaphoreSlim_,
+                       };
+
+    await BufferSubmit.SendAsync(blockRequest,
+                                 token);
+
+    await queue_.Reader.WaitToReadAsync(CancellationQueueTaskSource.Token)
+                .ConfigureAwait(false);
+
+    return await queue_.Reader.ReadAsync(token)
+                       .ConfigureAwait(false);
+  }
+
+  /// <summary>
+  ///   The method submit with One serialized argument that will be already serialized for byte[] MethodName(byte[]
```

Comment:
same comment than previous one, but one version has already serialized parameter but other not